supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -72,6 +71,17 @@
 
 *See also*:
 
+[[raw-interpreted-program]]
+==== image:images/yes.png[yes] raw-interpreted program (noun)
+
+*Description*: _Raw-interpreted programs_ are programs that do not need to be compiled and are directly run by a language interpreter.
```

Comment:
thinking a lot now about keeping the term singular in all definitions...or making them plural, as long as we do the same thing each time. if we go singular (which i think is more intuitive), change this one thusly:

A _raw-interpreted program_ is a program that doesn't [/does not] need to be..../?